float32 min_x
float32 min_y
float32 width
float32 height
float32 conf
string class_name
int32 class_id
int32 detection_id